### PR TITLE
Fix part of #103: Fix lint issue in lib package

### PR DESCRIPTION
**Pull request by anandwana001**
## Explanation

Fix part of #103 

Fixed `.js files in lib` package

## Reviewer
@jameesjohn 

## Checklist
- [ ] I have successfully deployed my own instance of Oppiabot.
  - You can find instructions for doing this [here](https://github.com/oppia/oppiabot/wiki/Deploying-your-own-instance-of-the-oppiabot).
- [ ] I have manually tested all the changes made in this PR following the [manual tests matrix](https://github.com/oppia/oppiabot/wiki/Manual-Tests-Matrix).

### CODE REVIEW

File: lib/checkIssueAssignee.js [hunk 1]
```diff
@@ -78,10 +82,11 @@ const checkAssignees = async (context) => {
       await context.github.issues.removeAssignees(assigneeParam);
     }
   } catch (error) {
+    // eslint-disable-next-line no-console
```
Comment (anandwana001) [inline lib/checkIssueAssignee.js:85]: this is done to disable the `no-console` issue. 
Should we update in .eslintrc directly?

File: lib/checkPullRequestLabels.js [hunk 2]
```diff
@@ -270,34 +275,35 @@ module.exports.checkChangelogLabel = async function(context) {
...
       const commentParams = context.issue({
         body:
           'Hi @' +
           user.login +
-          ', only members of the release team /cc @oppia/release-coordinators ' +
-          'are allowed to remove ' + DATASTORE_LABEL +' labels. ' +
+          ', only members of the release team ' +
+          '/cc @oppia/release-coordinators ' +
+          'are allowed to remove ' + DATASTORE_LABEL + ' labels. ' +
           'I will be adding it back. Thanks!',
```
Comment (vojtechjelinek) [inline lib/checkPullRequestLabels.js:307]:
```
'Hi @' + user.login + ', only members of the release team' +
'/cc @oppia/release-coordinators ' +
'are allowed to remove ' + DATASTORE_LABEL + ' labels. ' +
'I will be adding it back. Thanks!',
```

File: lib/checkPullRequestReview.js [hunk 3]
```diff
@@ -28,7 +28,8 @@ const PTAL_COMMENTS = ['PTAL', 'Please take a look'];
  * 2. Fetch the pull request for new changes.
  * 3. Check if the pull request has been closed or merged and exit.
  * 4. Check if the reviewer is still assigned to the PR and unassign them.
- * 5. Assign / ping the PR author if not already assigned (in the 3 minutes wait).
+ * 5. Assign / ping the PR author if not already assigned
+ * (in the 3 minutes wait).
```
Comment (vojtechjelinek) [inline lib/checkPullRequestReview.js:32]:
```suggestion
 * 5. Assign / ping the PR author if not already assigned
 *    (in the 3 minutes wait).
```

**Review by vojtechjelinek — CHANGES_REQUESTED** [review]
Thanks! Left a few comments.

**Review by jameesjohn — APPROVED** [review]
Thanks, @anandwana001!
I have no other comments apart from the ones mentioned by @vojtechjelinek.

File: lib/checkCriticalPullRequest.js [hunk 4]
```diff
@@ -25,7 +26,10 @@ const {
   pingAndAssignUsers
 } = require('./utils');
 
-const { teamLeads, oppiaMaintainers, SERVER_JOBS_ADMIN } = require('../userWhitelist.json');
+const { teamLeads,
+  oppiaMaintainers,
+  SERVER_JOBS_ADMIN
+} = require('../userWhitelist.json');
```
[the remark this answers is not in the document]
Comment (anandwana001) [reply lib/checkCriticalPullRequest.js:32]: Done

File: lib/checkMergeConflicts.js [hunk 5]
```diff
@@ -12,7 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-const { sleep, getAllOpenPullRequests, pingAndAssignUsers } = require('./utils');
+const { sleep,
+  getAllOpenPullRequests,
+  pingAndAssignUsers
+} = require('./utils');
```
[the remark this answers is not in the document]
Comment (anandwana001) [reply lib/checkMergeConflicts.js:18]: done

File: lib/checkPullRequestLabels.js [hunk 6]
```diff
@@ -187,8 +187,8 @@ module.exports.checkChangelogLabel = async function(context) {
   // eslint-disable-next-line no-console
   console.log(
     'RUNNING CHANGELOG LABEL CHECK ON PULL REQUEST ' +
-      pullRequestNumber +
-      ' ...'
+    pullRequestNumber +
+    ' ...'
```
[the remark this answers is not in the document]
Comment (anandwana001) [reply lib/checkPullRequestLabels.js:191]: done

File: lib/checkPullRequestLabels.js [hunk 7]
```diff
@@ -199,7 +199,8 @@ module.exports.checkChangelogLabel = async function(context) {
   if (hasChangelogLabel) {
     var changelogLabelIsValid = matchChangelogLabelWithRegex(changelogLabel);
     if (!changelogLabelIsValid) {
-      // There is some problem with the changelog label. Ping the dev workflow team
+      // There is some problem with the changelog label.
+      // Ping the dev workflow team
       // in a comment on the PR.
```
[the remark this answers is not in the document]
Comment (anandwana001) [reply lib/checkPullRequestLabels.js:204]: done

File: lib/checkPullRequestLabels.js [hunk 8]
```diff
@@ -243,10 +244,14 @@ module.exports.checkChangelogLabel = async function(context) {
...
-        ' -- could one of you please add the appropriate changelog label '+
+        ' -- could one of you please add the appropriate changelog label ' +
         'to this pull request? Thanks!';
     }
-    await pingAndAssignUsers(context, pullRequest, reviewersToPing, commentBody)
+    await pingAndAssignUsers(context,
+      pullRequest,
+      reviewersToPing,
+      commentBody
+    );
```
[the remark this answers is not in the document]
Comment (anandwana001) [reply lib/checkPullRequestLabels.js:254]: done

File: lib/checkPullRequestLabels.js (hunk 2 above)
Comment (anandwana001) [reply lib/checkPullRequestLabels.js:307]: done, Thanks for the comment.

File: lib/checkPullRequestReview.js (hunk 3 above)
Comment (anandwana001) [reply lib/checkPullRequestReview.js:32]: done

File: lib/checkIssueAssignee.js (hunk 1 above)
Comment (vojtechjelinek) [reply lib/checkIssueAssignee.js:85]: @anandwana001 Could you disable this in `.eslintrc` and remove all this disables?

Comment (anandwana001) [reply lib/checkIssueAssignee.js:85]: ok

Comment (anandwana001) [reply lib/checkIssueAssignee.js]: Done

File: lib/periodicChecks.js [hunk 9]
```diff
@@ -80,12 +80,14 @@ const ensureAllPullRequestsAreAssigned = async (context) => {
     );
 
     if (hasMergeConflict(pullRequestData)) {
+      // eslint-disable-next-line no-console
```
Comment (vojtechjelinek) [inline lib/periodicChecks.js:83]: Remove the `eslint-disable-next-line no-console` from this file.

Comment (anandwana001) [reply lib/periodicChecks.js]: ah, missed it. Done

**Review by vojtechjelinek — APPROVED** [review]
LGTM!

**Pull request merged**